doc/ddc-path.txt

```diff
@@ -60,6 +60,21 @@ cmd		(string[])
 
 	Default: ["find", "-maxdepth", "5"]
 
+						 *ddc-path-param-dirSeparator*
+dirSeparator	("slash" | "backslash" | "")
+		{for MS-Windows}
+	Change the directory separator to a forward or backward slash.
+
+	  slash		A forward slash is used.
+	  backslash	A backward slash is used.
+	  (empty)	Auto detect by:
+			- When |'completeslash'| is not empty, the value is
+			  used for this option.
+			- When |'shellslash'| is enabled, 'slash' is used.
+			- When |'noshellslash'|, 'backslash' is used.
```

Comment:
I am not sure. Is it possible to jump to the section declared the tags with quotations `'`. I guess we can just use `|completeslash|` for example.

Reply:
I made a mistake. References to options should enclose in single quotes only. eg.:

    'shellslash' (correct)
    |'shellslash'| (incorrect)
    |shellslash| (incorrect)

And, string should enclose in double quotes, because it is confusion to option.

Reply:
Noted. Thank you.